.gitignore

```diff
@@ -150,3 +150,5 @@ model_training/*data*
 model_training/tekton/secret
 local-dev-cluster
 tmp
+tests/db-models
+db-models
```

Comment:
Shall we use this PR to start keeping all tmp files for tests under `tmp/tests` directory?